rollouts/controllers/rollout_controller.go

```diff
@@ -680,13 +702,55 @@ func (r *RolloutReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if err := r.store.Init(); err != nil {
 		return err
 	}
-	// TODO: watch cluster resources as well
+
+	var containerCluster gkeclusterapis.ContainerCluster
+	containerCluster.SetGroupVersionKind(kccClusterGVK)
```

Comment:
nit: if kcctypes are added the scheme (which I believe otherwise ListClusters would have failed), this line is redundant :)